ci-operator/jobs/openshift/knative-build/openshift-knative-build-periodics.yaml

```diff
@@ -76,7 +76,7 @@ periodics:
         valueFrom:
           configMapKeyRef:
             key: openshift-knative-build-release-0.2.yaml
-            name: ci-operator-misc-configs
+            name: ci-operator-master-configs
```

Comment:
Aaaand these should be in `-misc-`, I think :)

https://github.com/openshift/release/blob/master/cluster/ci/config/prow/plugins.yaml#L627

Reply:
my `sed` fu is so bad